src/data/missions.csv

```diff
@@ -0,0 +1,7 @@
+"text","translation"
```

Comment:
```suggestion
"text", "translation"
```

Reply:
Currently the script relies on the space to skip non-translation text, it's bad and probably not obvious